server/lib/notifications/agents/webpush.ts

```diff
@@ -0,0 +1,205 @@
+        return {
+          notificationType: Notification[type],
+          subject: payload.subject,
+          message: payload.message,
+        };
+      case Notification.MEDIA_APPROVED:
+        return {
+          notificationType: Notification[type],
+          subject: payload.subject,
+          message: 'Your request has been approved.',
+          image: payload.image,
+          mediaType: payload.media?.mediaType,
+          tmdbId: payload.media?.tmdbId,
+          requestId: payload.request?.id,
+          actionUrl: `/${payload.media?.mediaType}/${payload.media?.tmdbId}`,
+        };
+      case Notification.MEDIA_AUTO_APPROVED:
+        return {
+          notificationType: Notification[type],
+          subject: payload.subject,
+          message: `${
+            payload.media?.mediaType === MediaType.MOVIE ? 'Movie' : 'Series'
+          } request was auto-approved.\nRequested by: ${
+            payload.request?.requestedBy.displayName
+          }`,
+          image: payload.image,
+          mediaType: payload.media?.mediaType,
+          tmdbId: payload.media?.tmdbId,
+          requestId: payload.request?.id,
+          actionUrl: `/${payload.media?.mediaType}/${payload.media?.tmdbId}`,
+        };
+      case Notification.MEDIA_AVAILABLE:
+        return {
+          notificationType: Notification[type],
+          subject: payload.subject,
+          message: 'Your request is now available.',
+          image: payload.image,
+          mediaType: payload.media?.mediaType,
+          tmdbId: payload.media?.tmdbId,
+          requestId: payload.request?.id,
+          actionUrl: `/${payload.media?.mediaType}/${payload.media?.tmdbId}`,
+        };
+      case Notification.MEDIA_DECLINED:
+        return {
+          notificationType: Notification[type],
+          subject: payload.subject,
+          message: 'Your request was declined.',
+          image: payload.image,
+          mediaType: payload.media?.mediaType,
+          tmdbId: payload.media?.tmdbId,
+          requestId: payload.request?.id,
+          actionUrl: `/${payload.media?.mediaType}/${payload.media?.tmdbId}`,
+        };
+      case Notification.MEDIA_FAILED:
+        return {
+          notificationType: Notification[type],
+          subject: payload.subject,
+          message: 'The request failed.',
+          image: payload.image,
+          mediaType: payload.media?.mediaType,
+          tmdbId: payload.media?.tmdbId,
+          requestId: payload.request?.id,
+          actionUrl: `/${payload.media?.mediaType}/${payload.media?.tmdbId}`,
+        };
+      case Notification.MEDIA_PENDING:
+        return {
+          notificationType: Notification[type],
+          subject: payload.subject,
+          message: `New ${
+            payload.media?.mediaType === MediaType.MOVIE ? 'movie' : 'series'
+          } request pending approval.\nRequested by: ${
+            payload.request?.requestedBy.displayName
+          }`,
```

Comment:
```suggestion
          message: `New ${
            payload.media?.mediaType === MediaType.MOVIE ? 'movie' : 'series'
          } request from ${payload.request?.requestedBy.displayName} pending approval.`,
```
I still strongly feel that we should not have a newline in this notification message.  We cannot guarantee that the first line will not wrap, and when it does, it will just result in the "requested by" information not being visible at all if only two lines of text are allowed.